modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -9,7 +9,20 @@
 
 To create CSI-provisioned persistent volumes that mount to these supported storage assets, {product-title} installs the necessary CSI driver Operator, the CSI driver, and the required storage class by default. For more details about the default namespace of the Operator and driver, see the documentation for the specific CSI Driver Operator.
 
-The following table describes the CSI drivers that are installed with {product-title} and which CSI features they support, such as volume snapshots, cloning, and resize.
+ifndef::openshift-rosa[]
+[IMPORTANT]
+====
+The AWS EFS and GCP Filestore CSI drivers are not installed by default, and must be installed manually. For instructions on installing the AWS EFS CSI driver, see link:https://access.redhat.com/documentation/en-us/openshift_dedicated/4/html/storage/using-container-storage-interface-csi#osd-persistent-storage-aws-efs-csi[Setting up AWS Elastic File Service CSI Driver Operator]. For instructions on installing the GCP Filestore CSI driver, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/storage/using-container-storage-interface-csi#persistent-storage-csi-google-cloud-file-overview[Google Compute Platform Filestore CSI Driver Operator].
+====
+endif::openshift-rosa[]
+The following table describes the CSI drivers that are
+ifndef::openshift-dedicated[]
+installed with {product-title}
+endif::openshift-dedicated[]
+ifndef::openshift-rosa[]
+supported by {product-title}
+endif::openshift-rosa[]
+and which CSI features they support, such as volume snapshots, cloning, and resize.
 
```

Comment:
Out of scope of this PR, but shouldn't "inline" be "Inline" in the column title for "inline ephemeral volumes"? Wondering if you have bandwidth to fix that if it is wrong.

Reply:
Good catch, pretty sure I can fix that issue. Thanks!